hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -170,6 +170,64 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
                 </div>
             </div>
 
+            <hr />
+            <div class="form-group">
+                <label class="col-md-3 control-label">Filter</label>
+                <div class="col-md-9">
+                    <div class="panel">
```

Comment:
No need for "panel" class (it adds some shadow, borders and radius). Or maybe just remove that div, I didn't try it.